### DISCUSSION TITLE
I lost my email account connected to my principal account

### DISCUSSION THREAD

**Discussion by no1crate2** [community/community#27423] [New to GitHub] [answered]
I was trying to connect to my account, but it ask me for a code in a email that I lost. I tried to acces to it by a lot of password and trying to contact Google Support for getting the account back, I didnt get any response, so Im trying to get any help by GitHub. There is any way to get my account back?

**Comment by jfullstackdev** [comment, marked as answer]
try requesting it here too 

<https://support.github.com/contact/account>

**Reply by queenofcorgis1** [reply]
@jdevstatic is correct - for account-specific situations, our Support team are the ones who can best assist!
